modules/core_project_factory/scripts/preconditions.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
I'm not sure how folks will feel about this, but I could not figure out a way to extract a parameter from the params destined for the python script without relying on external dependencies (sed, grep) unless it runs in bash, which allows arrays.